ironfish/src/wallet/walletdb/walletdb.test.ts

```diff
@@ -47,6 +55,126 @@ describe('WalletDB', () => {
     })
   })
 
+  describe('loadUnspentNotesByValue', () => {
+    it('notes should be returned in order of value', async () => {
+      const { node } = await nodeTest.createSetup()
```

Comment:
this should not be called in a test, it needs to be called in the describe or bugs can occur because it hooks into jests's beforeEach and afterAll.